app-utils/nuclei/autobuild/build

```diff
@@ -3,5 +3,5 @@ go build -v -o "nuclei" "$SRCDIR"/cmd/nuclei
 
 abinfo "Installing nuclei ..."
 install -Dvm755 "$SRCDIR"/nuclei -t "$PKGDIR"/usr/bin
-install -Dvm644 "$SRCDIR"/README*.md -t "$PKGDIR"/usr/share/doc/nuclei
-install -Dvm644 "$SRCDIR"/LICENSE.md "$PKGDIR"/usr/share/doc/nuclei/LINCENSE.md
+install -Dvm644 "$SRCDIR"/README*.md "$SRCDIR"/LICENSE.md \
+    -t "$PKGDIR"/usr/share/doc/nuclei/
```

Comment:
Add a final newline. It seems that your editor didn't follow the [editorconfig](https://editorconfig.org/), you may install a corresponding plugin/extension for your editor or use the editors listed on the website that support editorconfig.